Library/Homebrew/rubocops/bottle_block_cop.rb

```diff
@@ -0,0 +1,36 @@
+module RuboCop
+  module Cop
+    module CustomCops
```

Comment:
@GauthamGoli, this isn't really a problem as it's in another namespace. The only thing to look out for would be to reference the top-level `module Homebrew` with `::Homebrew` inside `Rubocop::Cop`, but I think we don't even need it in this context.